README.md

```diff
@@ -23,6 +24,37 @@ popd
 hugo serve -D
 ```
 
+### Docker
+For local development a docker image can be build and use.
+
+#### Build
+
+In order to build the image, just run a normal docker build command.
+
+```
+$ docker build -t myuser/ec2-spot-workshops:dev .
+Sending build context to Docker daemon   84.4MB
+Step 1/16 : FROM golang:alpine AS build
+*snip*
+Successfully tagged myuser/ec2-spot-workshops:dev
+```
+
+The image exposes the static website via port 1313.
```

Comment:
Nice :) . Thanks Christian 👍 

Just one comment : Is there any particular reason why we leave a reference to docker swarm service creation here?  

To simplify, I'd suggest we should remove the `Docker->build` section and just focus on the `Docker->development` section to provide a dev setup that people can set in "watch" mode for testing when they build content, something like:

### Development using Docker
For local development a docker image can be build using [docker-compose](https://docs.docker.com/compose/).
The following command will spin up a container exposing the website at [localhost:1313](http://localhost:1313) and 
mount `config.toml` and the directories `./content`, `./layouts` and `./static`, so that local changes will automatically be picked up by the development container.

```
$ docker-compose up
Starting ec2-spot-workshops_hugo_1 ... done
```

Reply:
Thanks for the feedback! I removed the Build piece.